docs/ble_hid.md

```diff
@@ -2,11 +2,9 @@
 Bluetooth connections help clean up the wire mess!
 
 ## CircuitPython
-If not running KMKPython, this does require the adafruit_ble library from Adafruit.
-This can be downloaded
-[here](https://github.com/adafruit/Adafruit_CircuitPython_BLE/tree/master/adafruit_ble).
-It is part of the [Adafruit CircuitPython Bundle](https://github.com/adafruit/Adafruit_CircuitPython_Bundle).
-Simply put this in the "root" of your CircuitPython device. If unsure, it's the folder with main.py in it, and should be the first folder you see when you open the device.
+If not running KMKPython, this does require the [adafruit_ble library from Adafruit](https://github.com/adafruit/Adafruit_CircuitPython_BLE/tree/master/adafruit_ble).
+It's part of the Adafruit CircuitPython Bundle, which can be downloaded [here](https://github.com/adafruit/Adafruit_CircuitPython_Bundle/releases/tag/20220715).
```

Comment:
1. please don't use `[here]()` links. Link name should reflect where it's pointing to.
2. please don't link to specific versions, unless necessary and stated so in the documentation. We just recently had issues with a link to a very old third-party library version, which caused a lot of confusion and was very hard to troubleshoot.